packages/lexical-playground/src/Editor.tsx

```diff
@@ -257,6 +260,9 @@ export default function Editor(): JSX.Element {
             maxLength={5}
           />
         )}
+        {(isCharCount || isCharCountUtf8) && (
+          <CharacterCountPlugin charset={isCharCount ? 'UTF-16' : 'UTF-8'} />
```

Comment:
This is what it would look like when the output of the plugin is headless

```suggestion
          <span className="character-count">
            <CharacterCountPlugin charset={isCharCount ? 'UTF-16' : 'UTF-8'} />
          </span>
```